cloud/vsphere/pods.go

```diff
@@ -32,8 +32,8 @@ import (
 	"sigs.k8s.io/cluster-api/cloud/vsphere/config"
 )
 
-var apiServerImage = "gcr.io/k8s-cluster-api/cluster-apiserver:0.0.4"
-var controllerManagerImage = "gcr.io/k8s-cluster-api/controller-manager:0.0.4"
+var apiServerImage = "gcr.io/k8s-cluster-api/cluster-apiserver:0.0.5"
```

Comment:
Did you mean to change the apiserver version for just vsphere? Would you need to update the makefile too?

Reply:
updated the apiserver as that is what is being used for gce and terraform.

Reply:
Oh I didn't see the others were already at 0.0.5, oops.